agent_touch.view.lkml

```diff
@@ -0,0 +1,91 @@
+view: agent_touch {
+  sql_table_name: dbt_arthur_common.zendesk_ticket_touch ;;
+
+
+  dimension: id {
+    description: "ID for the touch of a ticket"
+    primary_key: yes
+    type: number
+    sql: ${TABLE}."id" ;;
+  }
```

Comment:
Will anyone ever need this on its own? I think we can probably leave this hidden.